tools/backups2datalad/datasetter.py

```diff
@@ -178,12 +188,19 @@ def update_github_metadata(
             ds = Dataset(self.target_path / d.identifier)
             repo = self.get_github_repo_for_dataset(ds)
             log.info("Setting metadata for %s ...", repo.full_name)
-            stats = self.get_dandiset_stats(ds)
+            stats, substats = self.get_dandiset_stats(ds)
             repo.edit(
                 homepage=f"https://identifiers.org/DANDI:{d.identifier}",
                 description=self.describe_dandiset(d, stats),
             )
             ds_stats.append(stats)
+            if self.config.zarr_gh_org is not None:
+                for zarr_id, zarr_stat in substats.items():
+                    zarr_repo = self.get_github_repo(
+                        f"{self.config.zarr_gh_org}/{zarr_id}"
+                    )
+                    log.info("Setting metadata for %s ...", zarr_repo.full_name)
+                    zarr_repo.edit(description=self.describe_zarr(zarr_stat))
```

Comment:
overall feels like identical code added above in another function -- I didn't play the game of finding differences.  May be could be centralized into a helper and avoid code duplication?

Reply:
Done: 75b9615